langchain/llms/anyscale.py

```diff
@@ -102,7 +102,7 @@ def _call(
         """
 
         anyscale_service_endpoint = (
-            f"{self.anyscale_service_url}/{self.anyscale_service_route}"
+            f"{self.anyscale_service_url}{self.anyscale_service_route}"
```

Comment:
What happens if the user provides a URL with no trailing slash?

Reply:
Good point. Even though the provided `anyscale_service_url` and `anyscale_service_service` format should have correct format, I'm adding some validation code to ensure it